File: spec_backend_com_java_e_springboot/resolvidos/exercicio_27/alunos_media_01/src/Aluno.java
public class Aluno {
    // Atributos (variáveis de instância)
    public String nome;
    public int idade;
    public int matricula;
    public float media;

    // Método para cadastrar os dados básicos
    public void cadastrarDados(String nome, int idade, int matricula, float media) {
        this.nome = nome;
        this.idade = idade;
        this.matricula = matricula;
        this.media = media;
    }

    // Método para cadastrar a média de notas
    public void cadastrarMediaNotas(float media) {
        this.media = media;
    }

    // Método para exibir todas as informações
    public void exibirInformacoes() {
        System.out.println("=== DADOS DO ALUNO ===");
        System.out.println("Nome: " + nome);
        System.out.println("Idade: " + idade);
        System.out.println("Matrícula: " + matricula);
        System.out.println("Média de notas: " + media);
    }
}

File: mod_05_javascript_fundamentos_e_dom/exercicios/alunos_media_01/src/aluno.js
class Aluno {
    constructor() {
        this.nome = "";
        this.idade = 0;
        this.matricula = 0;
        this.media = 0.0;
    }

    cadastrarDados(nome, idade, matricula) {
        this.nome = nome;
        this.idade = idade;
        this.matricula = matricula;
    }

    cadastrarMediaNotas(media) {
        this.media = media;
    }

    exibirInformacoes() {
        console.log("\n=== DADOS DO ALUNO ===");
        console.log("Nome:", this.nome);
        console.log("Idade:", this.idade);
        console.log("Matrícula:", this.matricula);
        console.log("Média de notas:", this.media);
    }
}

module.exports = Aluno;